TruckMuncher/VendorMapViewController.swift

```diff
@@ -196,16 +197,45 @@ class VendorMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
     }
     
     func requestServingMode (isServing: Bool) {
+        var itemsOutOfStock = getOutOfStockItems()
+        if itemsOutOfStock.count > 0 && isServing == true {
+            var message = String()
+            for item in itemsOutOfStock {
+                message += "- " + (item as RMenuItem).name + "\n"
```

Comment:
If they have 10 items out of stock, this message is going to be way too long. I would limit it to maybe 3 items. And say `Continue with these items out of stock?` `Coca-Cola, Roast Beef, Tuna, and 4 others`.